src/components/course/data/hooks.jsx

```diff
@@ -789,13 +792,11 @@ export const useUserSubsidyApplicableToCourse = ({
     if (applicableUserSubsidy) {
```

Comment:
The innards of the function `setLearnerCreditSubsidy` needs to be run before this we get to this line. Otherwise this will always be null.